### PR TITLE
Add vuex support for editor activity

### PR DESCRIPTION
**Pull request by johan-bjareholt**
- Fixes editor support again so it works with other dates than today again (with vuex)
- Removed support for selecting a specific editor bucket
- Added back "duration" field for browser

Solves https://github.com/ActivityWatch/activitywatch/issues/316

### CODE REVIEW

File: src/store/modules/activity_daily.ts [hunk 1]
```diff
@@ -35,17 +35,26 @@ interface QueryOptions {
 const _state = {
   top_apps: [],
   top_titles: [],
+  app_chunks: [],
+
   top_domains: [],
   top_urls: [],
+  web_chunks: [],
```
Comment (ErikBjare) [inline src/store/modules/activity_daily.ts]: `app_chunks` and `web_chunks` aren't used anymore (because they make the query vastly more expensive).

Comment (johan-bjareholt) [reply src/store/modules/activity_daily.ts]: Done

**Comment by ErikBjare** [comment]
I haven't tested, but looks good to me. Merging.

**Comment by masaeedu** [comment]
Hello @johan-bjareholt. Do you know which version of aw-webui was this released in? Also, would you happen to know if there's a way I can check what version of aw-webui I'm running?

**Comment by ErikBjare** [comment]
aw-webui is distributed with aw-server, which in turn is distributed with
the ActivityWatch bundle. The version you're running should be visible in
the bottom right corner of the web UI.

Whichever release was made after the PR was merged should contain the fix,
if none such exists, it will be in the next one.

**Comment by johan-bjareholt** [comment]
> Whichever release was made after the PR was merged should contain the fix,
if none such exists, it will be in the next one.

@masaeedu  It is not in a release yet, so next version will have it!

**Pull request merged**